### PR TITLE
DOC-600 What's new RPCN

### PR DESCRIPTION
**Pull request by asimms41**
## Description

Resolves https://github.com/redpanda-data/documentation-private/issues/2498
Review deadline: 15 October 

NOTE: Links to connectors yet to be published will show as broken links.

## Page previews

[What's new](https://deploy-preview-72--redpanda-connect.netlify.app/redpanda-connect/whats_new_rpcn/)
## Checks

- [ ] New feature
- [x] Content gap
- [ ] Support Follow-up
- [ ] Small fix (typos, links, copyedits, etc)

### CODE REVIEW

File: modules/ROOT/pages/whats_new_rpcn.adoc [hunk 1]
```diff
@@ -0,0 +1,139 @@
...
+* The `timestamp` field to xref:components:outputs/kafka.adoc[`kafka`] and xref:components:outputs/kafka_franz.adoc[`kafka_franz`] outputs.
+* The `status_topic` field to the xref:components:redpanda/about.adoc[`redpanda` component].
+
+== Version 4.30.0 to 4.30.1
+
+Redpanda Connect now includes a beta version of the xref:components:inputs/splunk.adoc[`splunk` input].
+
+== Version 4.29.0
+
+Redpanda Connect now separates packages into enterprise and free bundles.
```
Comment (Feediver1) [inline modules/ROOT/pages/whats_new_rpcn.adoc:132]: Seems like we should explain what the difference between enterprise and free is here.

File: modules/ROOT/pages/whats_new_rpcn.adoc [hunk 2]
```diff
@@ -0,0 +1,139 @@
...
+Redpanda Connect has a new batch of beta AI processors for Self-Managed and Cloud platforms.
+
+=== For Self-Managed and Cloud platforms
+
+* xref:components:processors/openai_chat_completion.adoc[`openai_chat_completion`]
+* xref:components:processors/openai_embeddings.adoc[`openai_embeddings`]
+* xref:components:processors/openai_image_generation.adoc[`openai_image_generation`]
+* xref:components:processors/openai_speech.adoc[`openai_speech`]
+* xref:components:processors/openai_transcription.adoc[`openai_transcription`]
+* xref:components:processors/openai_translation.adoc[`openai_translation`]
```
Comment (Feediver1) [inline modules/ROOT/pages/whats_new_rpcn.adoc:102]: Thoughts on adding text that makes clear which require an enterprise license?

**Review by Feediver1 — APPROVED** [review]
Left a couple suggestions. Your call.

File: modules/ROOT/pages/whats_new_rpcn.adoc (hunk 1 above)
Comment (asimms41) [reply modules/ROOT/pages/whats_new_rpcn.adoc]: Added more details and linked to supported connectors page for more information.

**Pull request merged**